app/src/app/[lng]/[inventory]/preferences/activities/ACTIVITY_ITEMS.tsx

```diff
@@ -0,0 +1,26 @@
+export const ACTIVITY_ITEMS = [
```

Comment:
Would it make sense to move these constants like ACTIVITY_ITEMS and LINKS etc. into a single file? E.g. `constants.ts`.

Also, since there is no JSX in this file it's fine to name them .ts instead for clarity.